gallery/src/components/NxSmallTag/NxSmallTagExample.scss

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.gallery-overflow-example {
+  background-color: var(--nx-swatch-indigo-40);
```

Comment:
What's the purpose of this blue background?

Reply:
I think this can be removed if the overflow text should be hidden.